NETProvider/src/FirebirdSql.Data.FirebirdClient/Client/Native/IFbClient.cs

```diff
@@ -269,4 +351,98 @@ public interface IFbClient
 
 		#endregion
 	}
+
+	internal static class IFbClientEx
+	{
+		public static IntPtr isc_dsql_allocate_statement_ex(
```

Comment:
The method name can be without `_ex`, isn't it? The overload resolution will take care of that.